external/fv3fit/tests/test_sklearn_wrapper.py

```diff
@@ -52,28 +55,21 @@ def test_flatten_same_order():
     np.testing.assert_allclose(a, b)
 
 
-@pytest.fixture
-def test_regressor_ensemble():
```

Comment:
This fixture was only used by one test and its name added no additional information, so I merged the fixture into the test.